File: nagasenacs/src/Nagasena/Proc/Common/SchemaId.cs
namespace Nagasena.Proc.Common {

  /// <summary>
  /// SchemaId represents the <see href="http://www.w3.org/TR/exi/#key-schemaIdOption">schemaId</see> 
  /// property of an EXI stream.
  /// </summary>
  public sealed class SchemaId {

    private string m_value;

    public SchemaId(string val) {
      m_value = val;
    }

    public string Value {
      get {
        return m_value;
      }
    }

  }

}

File: nagasena/src/org/openexi/proc/common/SchemaId.java
package org.openexi.proc.common;

/**
 * SchemaId represents the <a href="http://www.w3.org/TR/exi/#key-schemaIdOption">schemaId</a> 
 * property of an EXI stream.
 */
public final class SchemaId {

  private String m_value;

  public SchemaId(String val) {
    m_value = val;
  }
  
  public String getValue() {
    return m_value;
  }

}
